src/beautifiers/rubocop.coffee

```diff
@@ -16,53 +18,50 @@ module.exports = class Rubocop extends Beautifier
       rubocop_path: true
   }
 
-  beautify: (text, language, options) ->
+  beautify: (text, language, options, context) ->
+    fullPath = context.filePath
+    [projectPath, relativePath] = atom.project.relativizePath(fullPath)
+
+    # Find the rubocop path
+    rubocopPath = "rubocop"
     @Promise.all([
       @which(options.rubocop_path) if options.rubocop_path
       @which('rubocop')
-    ]).then((paths) =>
+    ])
+    .then((paths) =>
       @debug('rubocop paths', paths)
-      _ = require 'lodash'
-      path = require 'path'
       # Get first valid, absolute path
-      rubocopPath = _.find(paths, (p) -> p and path.isAbsolute(p) )
+      rubocopPath = paths.find((p) -> p and path.isAbsolute(p))
```

Comment:
Since `if rubocopPath?` branch was removed below, I think this needs to be something like:

```coffeescript
rubocopPath = paths.find((p) -> p and path.isAbsolute(p)) or "rubocop"
```

such that the fallback value is `rubocop` [when the `find` returns `undefined`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find):

```coffeescript
rubocopPath = paths.find((p) -> p and path.isAbsolute(p)) or "rubocop"
# ...
rubocopPath = undefined or "rubocop"
# ...
rubocopPath = "rubocop"
```

And with this change I think you can also remove this line above:

```coffeescript
rubocopPath = "rubocop"
```

Reply:
Nice catch!